app/models/topic.rb

```diff
@@ -126,6 +126,11 @@ def assign(user_id=2, assigned_to)
     self.save
   end
 
+  def self.bulk_assign(post_attributes, assigned_to)
```

Comment:
Can we remove the old `assign` method now we're using this instead? It seems it's not used elsewhere (other than it tests).